modules/settings/assets/js/components/media-uploader/media-upload-check.js

```diff
@@ -0,0 +1,24 @@
+import CircularProgress from '@elementor/ui/CircularProgress';
+import { useSelect } from '@wordpress/data';
+
+export default function MediaUploadCheck(props) {
```

Comment:
To keep consistency:

```suggestion
export default const MediaUploadCheck = (props) => {
```